shinhye/controllers/userController.js

```diff
@@ -0,0 +1,36 @@
+const userService = require("../services/userService");
+
+const signUp = async (req, res) => {
+  try {
+    const { email, profileImage, password } = req.body;
+    if (!email || !password) {
+      const err = new Error({ message: "KEY_ERROR" });
+      err.statusCode = 400;
+      throw err;
+    }
+
+    await userService.signUp(email, profileImage, password);
+    return res.status(201).json({ message: "SIGNUP_SUCCESS" });
+  } catch (err) {
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    if (!email || !password) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await userService.login(email, password);
+    let token = userService.login(jwtToken);
```

Comment:
동일한 코드가 반복되고 있네요!
token이라는 변수에 27번 코드 라인의 결과 값을 바로 할당해주면 될 것 같습니다!

```suggestion
    let token = await userService.login(email, password);
```